tests/script_errors/sub_overflow/error.txt

```diff
@@ -0,0 +1,3 @@
+Error: 
+    2 | 0 - unsigned::max;
+      | ^^^^^^^^^^^^^^^^^ The binary operation `-` caused an overflow
```

Comment:
should we call it an `underflow` in this case? I think it might make it cleaner in cases where the values are not immediately obvious

Reply:
I thought about it, we  totally could but that'd introduce a whole new error class and means separating out substractions and additions,not sure if the benefits justify that.

Reply:
yup, I think it's a good compromise between ease of implementation and ease of use